lib/board.rb

```diff
@@ -0,0 +1,61 @@
+require 'pry'
+class Board
+
+
+	attr_accessor :cells, :edges, :corners, :center
+
+	def initialize
+		@cells = Array.new(9," ")
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.